libraries/anaconda3-buildtime/context/build

```diff
@@ -2,9 +2,9 @@
 
 set -e
 
-CONDAENV=conda_environment
+ANACONDA_ENV=/home/algo/conda_environment
 ROOTDIR=/opt/algorithm
 
 . /opt/conda/etc/profile.d/conda.sh
 
-conda env create -n $CONDAENV --file=$ROOTDIR/requirements.yaml -q --force
+conda env update --prefix ANACONDA_ENV --file=$ROOTDIR/requirements.yml
```

Comment:
Did you mean `$ANACONDA_ENV`? Now, I believe you are using the string `ANACONDA_ENV`. I'm not sure how things are working - maybe `~/conda_environment` is the default location for the dependencies?
Why did we have the force before?